src/features/app/components/ErrorBoundary/ErrorBoundary.js

```diff
@@ -8,7 +8,7 @@ import {
   Legend,
   ReportButton,
   Wrapper,
-} from 'components/ErrorBoundary/ErrorBoundary.styles';
+} from 'features/app/components/ErrorBoundary/ErrorBoundary.styles';
```

Comment:
Remind me of the conclusion about the resolution of the routes when there are in the same folder, it would be like this? Or should this be `./ErrorBoundary.styles`?

Reply:
i refactored this through `webstorm` which uses or absolute or relative paths.

Thats the reason behind

Reply:
Ok, so this is a WIP?